person.ru

```diff
@@ -1,15 +1,17 @@
 require 'corrector'
+require 'rental'
 
 class Person
   attr_accessor :name, :age
-  attr_reader :id
+  attr_reader :id, :rentals
 
```

Comment:
- [ ] Since a person can create many rentals, Getters and Setters for @rentals should be present`